base/logging_config.go

```diff
@@ -338,13 +304,21 @@ func validateLogFileOutput(logFileOutput string) error {
 
 	// Validate containing directory
 	logFileOutputDirectory := filepath.Dir(logFileOutput)
-	err := validateLogFilePath(logFileOutputDirectory)
+	// Make full directory structure if it doesn't already exist
+	err := os.MkdirAll(logFileOutputDirectory, 0700)
 	if err != nil {
-		return err
+		return errors.Wrap(err, ErrInvalidLogFilePath.Error())
+	}
+
+	// Ensure LogFilePath is a directory. We'll check file permissions when it opens/creates the logfile.
+	if f, err := os.Stat(logFileOutputDirectory); err != nil {
+		return errors.Wrap(err, ErrInvalidLogFilePath.Error())
+	} else if !f.IsDir() {
+		return errors.Wrap(ErrInvalidLogFilePath, "not a directory")
 	}
 
 	// Validate given file is writeable
```

Comment:
```suggestion
	// Validate given file is writeable. If the file does not exist, this will open a zero sized log file intentionally to set permissions. The permissions will then be used by lumberjack. Otherwise, lumberjack will use 0600 permissions.
```